daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -69,18 +61,18 @@ message NodeCreate {
 }
 
 message NodeFetch {
-    string contract_id = 1;
+    reserved 1; // was contract_id
```

Comment:
Is always unset since SDK 1.0.0.

Reply:
Is this because of `contract_id_struct`?

Reply:
yes.